docs/install/assets/eventing.md

```diff
@@ -0,0 +1,11 @@
+# Knative Serving installation files (0.21 releases)
+
+| File name | Description | Dependencies|
+| --- | --- | --- |
+| eventing-core.yaml | Knative Serving core components. |  none |
+| eventing-crds.yaml | Knative Serving core CRDs. | none |
+| eventing-post-install.yaml | Optional jobs after installing eventing-core.yaml. | eventing-core.yaml |
+| eventing-sugar-controller.yaml | Optional reconciler that watches for labels and annotations on certain resources to inject eventing components. | eventing-core.yaml |
+| in-memory-channel.yaml | Components to configure In-Memory Channels. | eventing-core.yaml |
+| mt-channel-broker.yaml | Components to configure MT Channel Broker. | eventing-core.yaml |
```

Comment:
I would spell out `MT` people might not know the meaning of `MT` also for future language translations fewer acronyms is better

Reply:
I agree.

Reply:
`MT` is Multi-Tenant, just in case.
When I first saw it I incorrectly thought it was `Memory Transport` 😁

Reply:
Thanks, I'll spell it out in another commit